bolt-cli/src/commands/operators/eigenlayer.rs

```diff
@@ -145,6 +145,9 @@ impl EigenLayerSubcommand {
                 let signature = Bytes::from(signer.sign_hash_sync(&signature_digest)?.as_bytes());
                 let signature = SignatureWithSaltAndExpiry { signature, expiry, salt };
 
+                // Sanitize extra data removing quotes
+                let extra_data = extra_data.trim_matches('"').to_string();
```

Comment:
Could maybe also trim whitespace etc?

Reply:
Yep, leading and ending whitespaces are 99% unwanted

Reply:
> Could maybe also trim whitespace?

all of it?